app/controllers/admin/reports_controller.rb

```diff
@@ -25,12 +25,15 @@ def update
     def process_report
       case params[:outcome].to_s
       when 'resolve'
-        @report.update(action_taken_by_current_attributes)
+        @report.update!(action_taken_by_current_attributes)
+        log_action :dismiss, @report
```

Comment:
should probably be `:resolve` to match the existing terminology.

Reply:
My concern is that that *any* action "resolves" a report. In this line, it's a resolve without any *side effects* like silence/suspend. It'd be weird if we had "resolves" logged for these but not for the silence/suspend versions. Therefore, I used a more clear terminology, "dismiss", i.e. "resolve without doing anything"

Reply:
hmm. why *aren't* we logging both? I.e., suspending a user could be:

```
log_action :resolve, @report
log_action :suspend, @user
```

otherwise you'll have no idea what report a given action was taken in response to